integrationtest/pg/powcli/docker-compose-pow.yml

```diff
@@ -0,0 +1,56 @@
+version: '3.7'
```

Comment:
Similarish thing as used in integration testing.
But allows you to spin up the whole stack, and play with the CLI.
Basically that magic `make` that I did in the demo video, and then used the CLI locally.